### PR TITLE
seed the db with predefined tags 

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

This PR seeds the db with predefined tags
#### How should this be manually tested?

1. Start the server with `npm run dev`
2. In the logs you will see the predefined tags being inserted into the db
3. You can also use pgadmin to look for the tags on the tags table

#### What are the relevant issues?
We had to seed the db with predefined tags

### CODE REVIEW

**Comment by beblicarl** [comment]
@jocrah , Good afternoon boss, kindly review this for me, thank you

File: migrations/1665144863994_seed-db-with-predefined-tags.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+	pgm.sql(`INSERT into tags
```
Comment (jocrah) [inline migrations/1665144863994_seed-db-with-predefined-tags.js:5]: we can use one insert statement for all of them. kindly check out how to use one insert statement to enter multiple values

Comment (beblicarl) [reply migrations/1665144863994_seed-db-with-predefined-tags.js:5]: > we can use one insert statement for all of them. kindly check out how to use one insert statement to enter multiple values

i couldn't find anything on that but I will keep searching

Comment (beblicarl) [reply migrations/1665144863994_seed-db-with-predefined-tags.js:5]: > https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-insert-multiple-rows/

Done, thanks a lot

**Review by jocrah — APPROVED** [review]

**Pull request merged**